python/vyos/ifconfig/ethernet.py

```diff
@@ -417,12 +418,38 @@ def set_ring_buffer(self, rx_tx, size):
             print(f'could not set "{rx_tx}" ring-buffer for {ifname}')
         return output
 
+    def set_switchdev(self, enable):
+        ifname = self.config['ifname']
+        if not bool(re.match(r"^eth[0-9]+$", ifname)):
```

Comment:
This seems to be superfluous, as handeld in the XML part

https://github.com/vyos/vyos-1x/blob/current/interface-definitions/interfaces_ethernet.xml.in#L17

Reply:
agree, removed